theseus_gui/src/store/search.js

```diff
@@ -21,7 +22,7 @@ export const useSearch = defineStore('searchStore', {
   }),
   actions: {
     getQueryString() {
-      let andFacets = ['project_type:modpack']
+      let andFacets = [`project_type:${this.projectType}`]
```

Comment:
this won't work for datapacks. datapacks are mods with the `datapack` loader. unf you basically have to hardcode this.